lib/CXGN/Genotype/Protocol.pm

```diff
@@ -97,6 +107,14 @@ sub BUILD {
     $self->reference_genome_name($map_details->{reference_genome_name});
     $self->species_name($map_details->{species_name});
     $self->sample_observation_unit_type_name($map_details->{sample_observation_unit_type_name});
+
+    my $q = "SELECT create_date, description FROM nd_protocol WHERE nd_protocol_id = ?;";
```

Comment:
The nd_protocol table does not have a 'description' column yet. Is there a patch for it? Otherwise, this is breaking the detail page.

https://cassava-test.sgn.cornell.edu/breeders_toolbox/protocol/4

CXGN::Genotype::Protocol::BUILD(): DBI Exception: DBD::Pg::st execute failed: ERROR:  column "description" does not exist
LINE 1: SELECT create_date, description FROM nd_protocol WHERE nd_pr...
                            ^ [for Statement "SELECT create_date, description FROM nd_protocol WHERE nd_protocol_id = ?;" with ParamValues: 1='4'] at /usr/local/share/website/local-lib/lib/perl5/x86_64-linux-gnu-thread-multi/Class/MOP/Method.pm line 128